Assets/MRTK/SDK/Experimental/InteractiveElement/Examples/Scripts/ClickedStateMessage.cs

```diff
@@ -0,0 +1,33 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License
+
+using UnityEngine;
+
+namespace Microsoft.MixedReality.Toolkit.Experimental.InteractiveElement.Examples
```

Comment:
Don't examples go under Assets/MRTK/Examples/Experimental? This includes scripts.

Reply:
Talked to Catherine about this, here's her reply:

Yes, but in this case the examples need to be in this folder because the features are only supported in Unity 2019.  I do not want to have an example scene with errors and missing scripts if someone is using MRTK in Unity 2018

I mentioned this in the PR description for the example scene: Add experimental Interactive Element example scene + example scripts by CDiaz-MS · [Pull Request #9231 · microsoft/MixedRealityToolkit-Unity (github.com)](https://github.com/microsoft/MixedRealityToolkit-Unity/pull/9231)